charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -285,11 +357,6 @@ data:
             regex: (.+)
             target_label: __metrics_path__
             replacement: /api/v1/nodes/${1}/proxy/metrics
-        metric_relabel_configs:
```

Comment:
@jpweber what happens here if there are multiple platforms running under a single cluster? Will we start scraping all airflows, regardless of what platform deployed them?

Reply:
I think this might be confusing because of how the diff is displayed here. This removal is not part of the airflow job, but part of the `kubernetes-nodes` and `kubernetes-nodes-cadvisor` job. But it does say if we are not in a single namespace collect all the kubelet information. Which provides container metrics and some measure of node metrics.

Reply:
An alternate method here is to had some kind of `isCloud` flag and only open it up if we are in the cloud rather than relying on the single namespace variable.

Reply:
Ah true, I misread the diff! :grimacing:  This is prob good then. I could see a separate flag, but this might be good for now - up to you. If we go separate flag, you could give it a descriptive name, rather than `isCloud`. Generally we've followed the same pattern elsewehre.

Reply:
If you're ok with it I think this can provide useful data to support if we get to the point someday where support can get access to this data in enterprise clusters to help debug things.